pasteit/routes/api.js

```diff
@@ -9,7 +10,7 @@ router.get('/api', function(req, res) {
 router.route('/api/create')
     .post(function(req, res) {
         let hash = fn.newPaste(req);
-        res.json({paste_url: req.get('host') + '/p/' + hash});
```

Comment:
Do we still need to put the slashes in if we're using join?

Reply:
https://stackoverflow.com/questions/9756567/do-you-need-to-use-path-join-in-node-js 2nd answer states, that you can leave them out, I guess there is some kind of sanitizing function in the end.
Changed it, tested it, working!